PxtlCa.XmlCommentMarkDownGenerator/XmlToMarkDown.cs

```diff
@@ -63,7 +61,7 @@ public static string ToMarkDown(this XNode node, string assemblyName = null)
                     name = anchor ? "seeAnchor" : "seePage";
                 }
                 //treat first Param element separately to add table headers.
-                if (name == "param"
+                if (name.EndsWith("param")
```

Comment:
This change allows `<typeparam>` elements to get the same "header" treatment as regular parameters.